docs/developers/nft-tokens.md

```diff
@@ -0,0 +1,378 @@
+---
+id: nft-tokens
+title: NFT tokens
+---
+
+:::important
+NFTs aren't enabled yet on Elrond Mainnet. See testnet or devnet
+:::
+
+## **Introduction**
+
+The Elrond protocol introduces native NFT support by adding metadata and attributes on top of the already existing [ESDT](/developers/esdt-tokens).
+This way, one can issue a semi fungible token or a non fungible token which is quite similar to an ESDT, but has a few more attributes, such as a changeable URI. 
+Once owning a quantity of a NFT/SFT, users will have their data store directly under their account, inside the trie. All the fields available inside a NFT/SFT token are:
+
+- attributes
+- balance
+- creator
+- hash
+- name
+- properties
+- royalties
+- tokenIdentifier
+- URIs
```

Comment:
I think you're missing the "nonce" here.

Reply:
these are the fields exposed by the API. In further iterations, we could all the properties.